showcase/package-lock.json

```diff
@@ -1562,7 +1562,6 @@
       "integrity": "sha512-qEdR5Am5ldIT/XCfdwE42/3AR4j5RwHnAv6k3/4ezz8+rj8jlJo39BMFbFmH/Pvf2CG9NIuiNde7pBflg1vYbQ==",
       "dev": true,
       "requires": {
-        "@types/jasmine": "~3.4.1",
```

Comment:
Strange, for some reason `@types/jasmine` was removed... 

I'll re-generate the package-lock.json then, that will most likely fix this kind of issues...